### DISCUSSION TITLE
I create a tabs. I want to change TOC when I change tab but the the TOC renders it all at once.

### DISCUSSION THREAD

**Discussion by ervinpiol** [fuma-nama/fumadocs#616] [Q&A]
![Uploading Screenshot 2024-07-21 185205.png…]()
Can I change TOC base on the tabs?

**Comment by fuma-nama** [comment]
Not supported. I think you shouldn't have headings under a tab since it will make in-page navigation unintuitive. Tab contents are conditionally rendered which we can no longer detect them with remark/rehype plugins
